Source/Core/Common/SettingsHandler.cpp

```diff
@@ -11,9 +11,10 @@
 #include <string>
 
 #ifdef _WIN32
+#include <windows.h> //NOLINT
+// The following Windows headers must be included AFTER windows.h.
```

Comment:
The problem is, I couldn't tell if that a simple "copy" would be enough, or if it wanted a real symlink, and I didn't dig far enough to find out.